ko/django_views/README.md

```diff
@@ -25,7 +25,7 @@ def post_list(request):
     return render(request, 'blog/post_list.html', {})
 ```
 
-방금 `post_list`라는 함수(`def`) 만들어 `요청(request)`을 넘겨받아 `render`메서드를 호출합니다. 이 함수는 호출하여 받은(return) `blog/post_list.html`템플릿을 보여줍니다.
+방금 `post_list`라는 함수(`def`)를 만들었습니다. 이 함수는 `요청(request)`을 넘겨받아 `render`메서드를 호출합니다. 이 함수는 `render` 메서드를 호출하여 받은(return) `blog/post_list.html`템플릿을 보여줍니다.
```

Comment:
[For Readability] Original Text:
`As you can see, we created a function (def) called post_list that takes request and will return the value it gets from calling another function render that will render (put together) our template blog/post_list.html.`